<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>491</width>
    <height>363</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QComboBox" name="cbComPorts">
   <property name="geometry">
    <rect>
     <x>11</x>
     <y>10</y>
     <width>91</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QComboBox" name="cbBaudRate">
   <property name="geometry">
    <rect>
     <x>111</x>
     <y>10</y>
     <width>71</width>
     <height>22</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>2400</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>4800</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>9600</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>19200</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>38400</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>57600</string>
    </property>
   </item>
  </widget>
  <widget class="QComboBox" name="cbPduFormat">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>10</y>
     <width>51</width>
     <height>22</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>8N2</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>8E1</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>8O1</string>
    </property>
   </item>
  </widget>
  <widget class="QPushButton" name="btnConnect">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>10</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Connect</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>72</y>
     <width>81</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Analog Input 1</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="tAnalogInput1">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>70</y>
     <width>113</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>100</y>
     <width>81</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Analog Input 2</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="tAnalogInput2">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>100</y>
     <width>113</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="btnRefresh">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>70</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Refresh</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="tDigitalInput1">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>130</y>
     <width>113</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>130</y>
     <width>81</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Digital input</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="tSlaveAddress">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>40</y>
     <width>111</width>
     <height>20</height>
    </rect>
   </property>
   <property name="layoutDirection">
    <enum>Qt::LeftToRight</enum>
   </property>
   <property name="text">
    <string>1</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QPushButton" name="btnReadSettings">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>330</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Read</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnWriteSettings">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>330</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Write</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnWriteROM">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>330</y>
     <width>101</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Write EEPROM</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnCoilPush">
   <property name="geometry">
    <rect>
     <x>209</x>
     <y>100</y>
     <width>160</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>Coil off</string>
   </property>
   <property name="checkable">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>6</x>
     <y>152</y>
     <width>471</width>
     <height>23</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QPushButton" name="btnState1">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="maximumSize">
       <size>
        <width>21</width>
        <height>21</height>
       </size>
      </property>
      <property name="text">
       <string>1</string>
      </property>
      <property name="checkable">
       <bool>true</bool>
      </property>
      <property name="checked">
       <bool>false</bool>
      </property>
      <property name="flat">
       <bool>false</bool>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="btnState2">
      <property name="maximumSize">
       <size>
        <width>21</width>
        <height>21</height>
       </size>
      </property>
      <property name="text">
       <string>2</string>
      </property>
      <property name="checkable">
       <bool>true</bool>
      </property>
      <property name="checked">
       <bool>false</bool>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="btnState3">
      <property name="maximumSize">
       <size>
        <width>21</width>
        <height>21</height>
       </size>
      </property>
      <property name="text">
       <string>3</string>
      </property>
      <property name="checkable">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="btnState4">
      <property name="maximumSize">
       <size>
        <width>21</width>
        <height>21</height>
       </size>
      </property>
      <property name="text">
       <string>4</string>
      </property>
      <property name="checkable">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="btnState5">
      <property name="maximumSize">
       <size>
        <width>21</width>
        <height>21</height>
       </size>
      </property>
      <property name="text">
       <string>5</string>
      </property>
      <property name="checkable">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="btnState6">
      <property name="maximumSize">
       <size>
        <width>21</width>
        <height>21</height>
       </size>
      </property>
      <property name="text">
       <string>6</string>
      </property>
      <property name="checkable">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="btnState7">
      <property name="maximumSize">
       <size>
        <width>21</width>
        <height>21</height>
       </size>
      </property>
      <property name="text">
       <string>7</string>
      </property>
      <property name="checkable">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="btnState8">
      <property name="maximumSize">
       <size>
        <width>21</width>
        <height>21</height>
       </size>
      </property>
      <property name="text">
       <string>8</string>
      </property>
      <property name="checkable">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="btnState9">
      <property name="maximumSize">
       <size>
        <width>21</width>
        <height>21</height>
       </size>
      </property>
      <property name="text">
       <string>9</string>
      </property>
      <property name="checkable">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="btnState10">
      <property name="maximumSize">
       <size>
        <width>21</width>
        <height>21</height>
       </size>
      </property>
      <property name="text">
       <string>10</string>
      </property>
      <property name="checkable">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="btnState11">
      <property name="maximumSize">
       <size>
        <width>21</width>
        <height>21</height>
       </size>
      </property>
      <property name="text">
       <string>11</string>
      </property>
      <property name="checkable">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="btnState12">
      <property name="maximumSize">
       <size>
        <width>21</width>
        <height>21</height>
       </size>
      </property>
      <property name="text">
       <string>12</string>
      </property>
      <property name="checkable">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="btnState13">
      <property name="maximumSize">
       <size>
        <width>21</width>
        <height>21</height>
       </size>
      </property>
      <property name="text">
       <string>13</string>
      </property>
      <property name="checkable">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="btnState14">
      <property name="maximumSize">
       <size>
        <width>21</width>
        <height>21</height>
       </size>
      </property>
      <property name="text">
       <string>14</string>
      </property>
      <property name="checkable">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="btnState15">
      <property name="maximumSize">
       <size>
        <width>21</width>
        <height>21</height>
       </size>
      </property>
      <property name="text">
       <string>15</string>
      </property>
      <property name="checkable">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="btnState16">
      <property name="maximumSize">
       <size>
        <width>21</width>
        <height>21</height>
       </size>
      </property>
      <property name="text">
       <string>16</string>
      </property>
      <property name="checkable">
       <bool>true</bool>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>194</y>
     <width>91</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Delay values, ms</string>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget_2">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>250</y>
     <width>468</width>
     <height>22</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_3">
    <item>
     <widget class="QDoubleSpinBox" name="sbDelayValue_9">
      <property name="frame">
       <bool>true</bool>
      </property>
      <property name="buttonSymbols">
       <enum>QAbstractSpinBox::NoButtons</enum>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QDoubleSpinBox" name="sbDelayValue_10">
      <property name="frame">
       <bool>true</bool>
      </property>
      <property name="buttonSymbols">
       <enum>QAbstractSpinBox::NoButtons</enum>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QDoubleSpinBox" name="sbDelayValue_11">
      <property name="frame">
       <bool>true</bool>
      </property>
      <property name="buttonSymbols">
       <enum>QAbstractSpinBox::NoButtons</enum>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QDoubleSpinBox" name="sbDelayValue_12">
      <property name="frame">
       <bool>true</bool>
      </property>
      <property name="buttonSymbols">
       <enum>QAbstractSpinBox::NoButtons</enum>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QDoubleSpinBox" name="sbDelayValue_13">
      <property name="frame">
       <bool>true</bool>
      </property>
      <property name="buttonSymbols">
       <enum>QAbstractSpinBox::NoButtons</enum>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QDoubleSpinBox" name="sbDelayValue_14">
      <property name="frame">
       <bool>true</bool>
      </property>
      <property name="buttonSymbols">
       <enum>QAbstractSpinBox::NoButtons</enum>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QDoubleSpinBox" name="sbDelayValue_15">
      <property name="frame">
       <bool>true</bool>
      </property>
      <property name="buttonSymbols">
       <enum>QAbstractSpinBox::NoButtons</enum>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QDoubleSpinBox" name="sbDelayValue_16">
      <property name="frame">
       <bool>true</bool>
      </property>
      <property name="buttonSymbols">
       <enum>QAbstractSpinBox::NoButtons</enum>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QComboBox" name="cbDevicePduFormat">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>280</y>
     <width>51</width>
     <height>22</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>8N2</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>8E1</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>8O1</string>
    </property>
   </item>
  </widget>
  <widget class="QComboBox" name="cbDeviceBaudRate">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>280</y>
     <width>71</width>
     <height>22</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>2400</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>4800</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>9600</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>19200</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>38400</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>57600</string>
    </property>
   </item>
  </widget>
  <widget class="QLabel" name="label_7">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>42</y>
     <width>81</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Slave address</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnRefreshAuto">
   <property name="geometry">
    <rect>
     <x>294</x>
     <y>70</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Over 200ms</string>
   </property>
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="autoRepeat">
    <bool>false</bool>
   </property>
   <property name="autoExclusive">
    <bool>false</bool>
   </property>
   <property name="autoRepeatDelay">
    <number>200</number>
   </property>
   <property name="autoRepeatInterval">
    <number>200</number>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>216</y>
     <width>468</width>
     <height>22</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_2">
    <item>
     <widget class="QDoubleSpinBox" name="sbDelayValue_1">
      <property name="frame">
       <bool>true</bool>
      </property>
      <property name="buttonSymbols">
       <enum>QAbstractSpinBox::NoButtons</enum>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QDoubleSpinBox" name="sbDelayValue_2">
      <property name="frame">
       <bool>true</bool>
      </property>
      <property name="buttonSymbols">
       <enum>QAbstractSpinBox::NoButtons</enum>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QDoubleSpinBox" name="sbDelayValue_3">
      <property name="frame">
       <bool>true</bool>
      </property>
      <property name="buttonSymbols">
       <enum>QAbstractSpinBox::NoButtons</enum>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QDoubleSpinBox" name="sbDelayValue_4">
      <property name="frame">
       <bool>true</bool>
      </property>
      <property name="buttonSymbols">
       <enum>QAbstractSpinBox::NoButtons</enum>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QDoubleSpinBox" name="sbDelayValue_5">
      <property name="frame">
       <bool>true</bool>
      </property>
      <property name="buttonSymbols">
       <enum>QAbstractSpinBox::NoButtons</enum>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QDoubleSpinBox" name="sbDelayValue_6">
      <property name="frame">
       <bool>true</bool>
      </property>
      <property name="buttonSymbols">
       <enum>QAbstractSpinBox::NoButtons</enum>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QDoubleSpinBox" name="sbDelayValue_7">
      <property name="frame">
       <bool>true</bool>
      </property>
      <property name="buttonSymbols">
       <enum>QAbstractSpinBox::NoButtons</enum>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QDoubleSpinBox" name="sbDelayValue_8">
      <property name="frame">
       <bool>true</bool>
      </property>
      <property name="buttonSymbols">
       <enum>QAbstractSpinBox::NoButtons</enum>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QLabel" name="label_8">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>283</y>
     <width>131</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Device connection settings</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
